app/controllers/buddies_controller.rb

```diff
@@ -2,7 +2,7 @@ class  BuddiesController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
 
   def index
-    @buddies = Buddy.all
+    @buddies = Buddy.all.page(params[:page]).per(1)
```

Comment:
Right now you're paginating 1 Buddy title per page. Is that what you want? Something like 10 titles might be more usable.